torch/_dynamo/eval_frame.py

```diff
@@ -1307,7 +1303,6 @@ def export(
 
     def inner(*args, **kwargs):
         combined_args = _combine_args(_f, args, kwargs)
-        _check_dynamic_shapes(combined_args, dynamic_shapes)
         constraints = _process_dynamic_shapes(combined_args, dynamic_shapes)
```

Comment:
Move this to _export_to_torch_ir

Reply:
Meaning what, the errors happen earlier or later?

Reply:
earlier, export_to_torch_ir() will check the original spec, and do the transform